examples/net_framework/README.md

```diff
@@ -0,0 +1,172 @@
+# Building DeepSpeech for Windows
+
+Now we can build the native DeepSpeech library and run inference on Windows using the C# client.
+
+**Table of Contents**
+
+- [Prerequisites](#prerequisites)
+- [Getting the code](#getting-the-code)
+- [Configuring the paths](#configuring-the-paths)
+- [Adding environment variables](#adding-environment-variables)
+    - [MSYS2 paths](#msys2-paths)
+    - [BAZEL path](#bazel-path)
+    - [Python path](#python-path)
+    - [CUDA paths](#cuda-paths)
+- [Building the code](#building-the-code)
+    - [Build for CPU](#cpu)
+    - [Build with CUDA support](#gpu-with-cuda)
+- [Using the generated library](#using-the-generated-library)
+## Prerequisites
+
+* [Python 3.6](https://www.python.org/)
+* [Git Large File Storage](https://git-lfs.github.com/)
+* [MSYS2](https://www.msys2.org/)
+* [Bazel v0.17.2](https://github.com/bazelbuild/bazel/releases)
+* [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk)
+* Windows 10
+* [Visual Studio 2017 Community](https://visualstudio.microsoft.com/vs/community/) 
+
+Inside the Visual Studio Installer enable `MS Build Tools` and `VC++ 2015.3 v14.00 (v140) toolset for desktop`
+
+If you want to enable CUDA support you need to install:
+
+* [CUDA 9.0 and cuDNN 7.3.1](https://developer.nvidia.com/cuda-90-download-archive) 
+
+
+## Getting the code
+
+We need to clone `mozilla/DeepSpeech` and `mozilla/tensorflow`.
+
+```bash
+git clone https://github.com/mozilla/DeepSpeech
+```
+
+```bash
+git clone https://github.com/mozilla/tensorflow
+```
+
+## Configuring the paths
+
+We need to create a symbolic link, for this example let's suppose that we cloned into `D:\cloned` and now the structure looks like:
+
+    .
+    ├── D:\
+    │   ├── cloned                 # Contains DeepSpeech and tensorflow side by side
+    │   │   ├── DeepSpeech         # Root of the cloned DeepSpeech
+    │   │   ├── tensorflow         # Root of the cloned Mozilla's tensorflow 
+    └── ...
+
+Change your path accordingly to your path structure, for the structure above we are going to use the following command:
+
+```bash
+mklink /d "D:\cloned\tensorflow\native_client" "D:\cloned\DeepSpeech\native_client"
+```
+
+## Adding environment variables
+
+After you install the requirements there are few environment variables that we need to add.
+
+#### MSYS2 paths
+
+For MSYS2 we need to add `C:\msys64\usr\bin` to the environment variables then we need to run:
+
+```bash
+pacman -Syu
```

Comment:
Where does `pacman` comes from? It's the ArchLinux package manager as much as I can tell.

Reply:
It's also the MSYS2 package manager.

Reply:
I'll add that we can run pacman after installing MSYS2.